src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -475,7 +475,7 @@ private function getNewOrder(array $uses, Tokens $tokens)
             // Ordering groups
             $sortedGroups = [];
             foreach ($this->configuration['imports_order'] as $type) {
-                if (isset($groupedByTypes[$type]) && !empty($groupedByTypes[$type])) {
+                if (isset($groupedByTypes[$type]) && [] !== $groupedByTypes[$type]) {
```

Comment:
Second part of the statement is actually completely unnecessary here. There is no reason not to pass empty array to foreach. It won't do anything for empty arrays.